RoboticonColony/Assets/Code/Roboticon.cs

```diff
@@ -38,30 +53,94 @@ public Roboticon(Tile selectedTile)
         bonusProduction[ItemType.Power] = 1;

```

Comment:
Production Multiplier defaults to 1, as 0 would result in no production, multiplication by 0 will always give 1, multiplying by 1 will mimic the tile's production.